### ISSUE TITLE
Bug: wrong render of projct names

### ISSUE THREAD

**Issue by sidkhuntia** [JammUtkarsh/pms#14]
![image](https://github.com/JammUtkarsh/pms/assets/77291662/46d8b394-0103-441b-8444-c084707c2037)

Steps to reproduce:
1. Add projects such that it results in pagination

**Comment by sidkhuntia** [comment]
I guess this was after you updated keys in fca6234171baf804a7d5d8c2da811cbc4cb85dd2.

**Comment by jammutkarsh** [comment]
No, this bug has persisted for the longest time. It is related to some rendering issue, which even I am not sure of why this is happening.

I plan to rewrite/refactor the entire app based based on the [template](https://github.com/charmbracelet/bubbletea-app-template) and taking the UI inspiration from [task timer](https://github.com/caarlos0/tasktimer).

It's a long task, so I plan to do this once I have settled.

Thanks for creating the issue though.
